test/functional/pdo_sqlsrv/MsCommon_mid-refactor.inc

```diff
@@ -168,7 +168,7 @@ class ColumnMeta
      */
     public function isEncryptableType()
     {
-        $unsupportedTypes = array("money", "smallmoney", "image", "ntext", "text", "xml", "sql_variant");
+        $unsupportedTypes = array("money", "smallmoney", "image", "ntext", "text", "xml", "sql_variant", "timestamp", "geography", "geometry", "hierarchyid");
         if (!$this->forceEncrypt && in_array(strtolower($this->dataType), $unsupportedTypes)) {
```

Comment:
Didn't realize you use the flag _forceEncrypt_. Under what conditions will this flag be true? How could we force encrypt a column type when AE is not supported for that type?

Reply:
there are a few tests for testing error returned from creating table containing some of these types